Slim/Web/Settings/Server/Behavior.pm

```diff
@@ -30,7 +30,7 @@ sub prefs {
 				conductorInArtists bandInArtists variousArtistAutoIdentification
 				ignoreReleaseTypes cleanupReleaseTypes groupArtistAlbumsByReleaseType
 				useTPE2AsAlbumArtist variousArtistsString ratingImplementation useUnifiedArtistsList
-				skipsentinel showComposerReleasesbyAlbum showComposerReleasesbyAlbumGenres)
+				skipsentinel showComposerReleasesbyAlbum showComposerReleasesbyAlbumGenres onlyAlbumYears)
```

Comment:
Please add the new pref to the "cache buster" list in https://github.com/LMS-Community/slimserver/blob/public/9.0/Slim/Web/XMLBrowser.pm#L43